frontend/static/css/form.css

```diff
@@ -24,7 +24,7 @@ noscript {
   display: block;
   margin: 2em 4em;
   font-size: 1.5em;
-  color: gray;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value shows that `gray` is `#808080`. If you also want to propose non-refactoring changes, would you mind putting them in a separate PR for easier review? Thanks! :-)

Reply:
This was a mistake, fixed.